compiler/test/suites/memory_base.re

```diff
@@ -0,0 +1,18 @@
+open Grain_tests.TestFramework;
+open Grain_tests.Runner;
+
+describe("memory_base", ({test}) => {
+  let assertFileRun = makeFileRunner(test);
+  // Non-snapshots for now, since we want to track functional issues.
+  // Maybe these can be migrated to snapshots once we're confident in
+  // everything being stable
+  assertFileRun("basecase", "memoryBase/basecase", "HelloWorld\n");
+  assertFileRun("no_op", "memoryBase/noop", "HelloWorld\n");
```

Comment:
`no_op` might not be a good name since it does have an effect, turning off some class of optimizations. Not sure what to call it though 🤔